packages/backend/package.json

```diff
@@ -45,6 +45,7 @@
         "dotenv": "^16.3.1",
         "fast-check": "^3.19.0",
         "kysely-codegen": "^0.11.0",
+        "next": "^14.2.5",
```

Comment:
Jim, we've talked about this. You can't add these dependencies so willy nilly. Just like we don't have a dependency on `express` we shouldn't have a dependency on Next.

The fact that you mark it as a `dev` dependency doesn't mean the bundler won't try to bundle it: it's part of production code now.